addon/run-task.ts

```diff
@@ -175,29 +175,36 @@ export function scheduleTask(
    @method throttleTask
    @param { Object } obj the instance to register the task for
    @param { String } taskName the name of the task to throttle
-   @param { Number } [timeout] the time in the future to run the task
+   @param { ...* } [throttleArgs] arguments to pass to the throttled method
+   @param { Number } timeout the time in the future to run the task
```

Comment:
The param that `throttle` expects as the `number` is called `spacing`, which indicates the interval when the function will be invoked. Can we keep this consistent by renaming `timeout` to `spacing`?